src/com/seafile/seadroid2/provider/SeafileProvider.java

```diff
@@ -56,17 +57,24 @@
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
 
 /**
  * DocumentProvider for the Storage Access Framework.
  *
- * It depends on API level 19.
+ * It depends on API level 19 and supports API level 21.
  *
  * This Provider gives access to other Apps to browse, read and write all files
  * contained in Seafile repositories.
  *
  */
-@TargetApi(Build.VERSION_CODES.KITKAT)
+@TargetApi(Build.VERSION_CODES.LOLLIPOP)
```

Comment:
the IDE like Intellij IDEA will coercively complain an error if changed the target API to 21, we should modify the README file in order other developers can build and run the project.